examples/word-count/dev/user.clj

```diff
@@ -0,0 +1,118 @@
+(ns user
+  (:require [clojure.java.shell :refer [sh]]
+            [clojure.string :as str]
+            [jackdaw.admin :as ja]
+            [jackdaw.repl :refer :all]
+            [jackdaw.serdes :as js]
+            [jackdaw.streams :as j]
+            [jackdaw.streams.describe :as jsd]
+            [integrant.core :as ig]
+            [integrant.repl :refer [clear go halt prep init reset reset-all]]
+            [word-count])
+  (:import [clojure.lang ILookup Associative]))
+
+
+(def config
+  {:streams-config {:application-id "word-count"
+                    :bootstrap-servers "localhost:9092"
+                    :default-key-serde "jackdaw.serdes.EdnSerde"
+                    :default-value-serde "jackdaw.serdes.EdnSerde"
+                    :cache-max-bytes-buffering "0"}
+
+   :topology {:topology-builder-fn :word-count/topology-builder}
+
+   :topics {:streams-config (ig/ref :streams-config)
+            :topology (ig/ref :topology)}
+
+   :app {:streams-config (ig/ref :streams-config)
+         :topology (ig/ref :topology)
+         :topics (ig/ref :topics)}})
+
+
+(integrant.repl/set-prep! (constantly config))
+
+
+(deftype FakeTopicMetadata []
+  ILookup
+  (valAt [this key]
+    {:topic-name (name key)
+     :partition-count 1
+     :replication-factor 1
+     :key-serde (js/edn-serde)
```

Comment:
I think that we should use a string serde for the key to make it easier to join with other topics. See https://github.com/FundingCircle/jackdaw-repl/commit/1ddd902e3ec325c862bdb2ef06f01050255b665e

Reply:
Keep in mind this is just for prototyping. We shouldn't use this or EDN keys in production.

Reply:
I was thinking we could add a new arity that takes an argument so the user can set their own default. Would that address your use case?